static/css/devhub/base.less

```diff
@@ -0,0 +1,97 @@
+@import '../impala/fonts/fira/fira.less';
+
+// Colors
+@primary-font: 'Fira Sans', sans-serif;;
+
+// Common styles
+section {
+  padding: 0 20px 60px 20px;
+
+  & > h1 {
+    font-size: 40px;
+    line-height: 1.2;
+    margin-bottom: 0;
+    margin-top: 20px;
+  }
+
+  & > p {
+    font-weight: 300;
+    margin-top: 10px;
+  }
+}
+
+.button {
+  padding: 10px 30px;
+  display: inline-block;
+  text-decoration: none;
+  text-align: center;
+  color: #fff;
+  border: 1px #979797;
+  border-radius: 2px;
+  box-shadow: 0 2px 3px 0 rgba(0, 0, 0, 0.3);
+
+  &.green {
+    background-color: #57BD35;
```

Comment:
Would be good to var as many colours as possible. Will not only make changes easier it will help to see that we don't end up with too many variants.